### ISSUE TITLE
[EN][Object] Maraudon Portal

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4291]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The portal is spawned but can not be used. It was stated to have been fixed but is only a unusable portal.
Closed report https://github.com/dalaranwow/dalaran-wow/issues/2283
**Expected behaviour**: Tell us what should happen instead
Go to the portal and spawn in.
**Steps to reproduce the problem**:

1. Go to portal in Maraudon
2. Try to use
3. 

**Include proofs for this behaviour**
![wowscrnshot_012217_172718](https://cloud.githubusercontent.com/assets/12252441/22186705/0c179876-e0c8-11e6-9ff1-2542a3901b4f.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
rev: https://github.com/dalaranwow/dalaran-wow/issues/3859

**Comment by Rushor** [comment]
DB/Gameobject: Portal to Inner Maraudon - should always be usable

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4291
* commit: 9555c29579d1122b55fcf17f61d61bb5ceb05f23

![GitHub Logo](http://i.imgur.com/UGOSy5Z.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
Already on PTR.
